hook.php

```diff
@@ -297,8 +298,22 @@ function plugin_escalade_install() {
             );
          }
       }
+      if ($DB->fieldExists('glpi_plugin_escalade_configs', 'remove_delete_supplier_btn')) {
+         $migration->changeField('glpi_plugin_escalade_configs',
+                                 'remove_delete_supplier_btn',
+                                 'remove_delete_assign_supplier_btn',
+                                 'bool',
+                                 ['value' => 1]);
+      }
```

Comment:
Is this a fix for a previous installation ?
It does not seems related to current PR. Please open another PR for this change if it has to be kept.